docs/install/install-gcp-broker.md

```diff
@@ -65,13 +69,14 @@ Workload Identity see
 
    ```shell
    gcloud iam service-accounts keys create cre-dataplane.json \
```

Comment:
Doesn't have to be this PR, but `cre-dataplane.json` should be renamed too.

Reply:
Renamed to `events-broker-key.json` or `events-sources-key.json` for broker and sources respectively.